### PR TITLE
quickstart: describe steps to run VM on VM Cloud

### PR DESCRIPTION
**Pull request by yatskevich**
### Describe Your Changes

Describe steps to run VictoriaMetrics Single node or Cluster on VictoriaMetrics Cloud
### Checklist

The following checks are **mandatory**:

- [x] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Comment by hagen1778** [comment]
@yatskevich please see update https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6877/commits/ea0ad1bb5d80257997bdc2858d184ad3659cd335

**Comment by hagen1778** [comment]
@tenmozes I need your approval for this.

File: docs/Quick-Start.md [hunk 1]
```diff
@@ -11,16 +11,18 @@ aliases:
 ---
 ## How to install
 
-VictoriaMetrics is distributed in two forms:
+VictoriaMetrics is distributed in the following forms:
```
Comment (yatskevich) [inline docs/Quick-Start.md:14]: I thought that versions of VictoriaMetrics are just Single-node and Cluster and this fact stays unchanged, so VM Cloud was included as an additional method (compared to Docker or k8s) of running Single-node or Cluster version.

**Review by tenmozes — APPROVED** [review]

File: docs/Quick-Start.md (hunk 1 above)
Comment (hagen1778) [reply docs/Quick-Start.md:14]: I see. From my perspective VM cloud is a separate, managed version of VM. And I think it is worth to list it among other versions

**Review by hagen1778 — APPROVED** [review]

**Pull request merged**